test/commands/ledger/reset.test.ts

```diff
@@ -39,20 +38,21 @@ describe('ledger:reset', () => {
 
     }))
 
-    test
-      .stdout()
-      .command(['ledger:reset', '-t', testToken])
-      .it('resets ledger fully', (ctx) => {
-        expect(ctx.stdout).to.contain('fully')
-      })
+  })
+
+  test
```

Comment:
Earlier I commented that all tests should be in a `it()` section, but I misunderstood what `test` was doing here. The `test` function already creates the test, so I messed up all tests to not run.

So this PR became a bit fatter because I had to fix that, and then also fix all tests that weren't actually passing.

Reply:
`before()`, nice!